src/encoded/schemas/quality_metric.json

```diff
@@ -0,0 +1,94 @@
+{
+    "description": "Schema for reporting the specific calculation of an quality metrics",
+    "id": "/profiles/quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step", "files"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/standard_status"}
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "step": {
+            "title": "Analysis Step",
+            "description": "Reference to analysis step run in pipeline",
+            "type": "string",
+            "linkTo": "analysis_step_run"
+        },
```

Comment:
Should this maybe be renamed to 'step_run' to avoid confusion with 'analysis_step'?